src/main/java/org/threeten/extra/chrono/InternationalFixedChronology.java

```diff
@@ -0,0 +1,484 @@
+ *    and/or other materials provided with the distribution.
+ *
+ *  * Neither the name of JSR-310 nor the names of its contributors
+ *    may be used to endorse or promote products derived from this software
+ *    without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+ * CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+ * EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+ * PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+ * PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
+ * LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ * NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package org.threeten.extra.chrono;
+
+import java.io.Serializable;
+
+import java.time.Clock;
+import java.time.DateTimeException;
+import java.time.ZoneId;
+
+import java.time.chrono.AbstractChronology;
+import java.time.chrono.Era;
+
+import java.time.temporal.ChronoField;
+import java.time.temporal.TemporalAccessor;
+import java.time.temporal.ValueRange;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * The International Fixed calendar system.
+ * <p>
+ * This chronology defines the rules of the International Fixed calendar system.
+ * It shares the leap year rule with the Gregorian calendar.
+ * Dates are aligned such that {@code 0001-01-01 (International Fixed)} is {@code 0001-01-01 (ISO)}.
+ * <p>
+ * This class is not proleptic.  It implements accepts only years greater or equal to 1.
+ * <p>
+ * This class implements a calendar where January 1st is the start of the year.
+ * <p>
+ * The fields are defined as follows:
+ * <ul>
+ * <li>era - There is only one era, the current 'Common Era' (CE).
+ * <li>year-of-era - The year-of-era for the current era increases uniformly from the epoch at year zero.
+ * <li>proleptic-year - The proleptic year is the same as the year-of-era for the current era.
+ * <li>month-of-year - There are 13 months in an International Fixed year, numbered from 1 to 13.
+ * <li>day-of-month - There are 28 days in an International Fixed month, numbered from 1 to 28.
+ * <li>day-of-year - There are 365 days in a standard International Fixed year and 366 days in a leap year.
+ *  The days are numbered accordingly.
+ * <li>leap-year - Leap years occur every 4 years, but skips 3 out of four centuries, i.e. when the century is not divisible by 400.
+ *  This is the same rule in use for the Gregorian calendar.
+ * <li>Week day - every month starts on a Sunday.  Leap-day and year-day are considered month-less Sundays.
```

Comment:
Leap-day and year-day are **not** Sunday.
Besides that, you need to document what month/day-of-month values the various getters return for those two days.